app/services/nomis/api/offender_api.rb

```diff
@@ -0,0 +1,81 @@
+module Nomis
+  module Api
+    class OffenderApi
+      extend Elite2Api
+
+      # rubocop:disable Metrics/MethodLength
+      def self.list(prison, page = 0, page_size: 10)
+        route = "/elite2api/api/locations/description/#{prison}/inmates"
+
+        page_meta = nil
+        page_offset = page * page_size
+        hdrs = paging_headers(page_size, page_offset)
+
+        data = e2_client.get(route, extra_headers: hdrs) { |json, response|
+          total_records = response.headers['Total-Records'].to_i
+          records_shown = json.length
+          page_meta = make_page_meta(
+            page, page_size, total_records, records_shown
+          )
+        }
+
+        offenders = data.map { |offender|
+          api_deserialiser.deserialise(Nomis::Models::OffenderShort, offender)
+        }
+
+        ApiPaginatedResponse.new(page_meta, offenders)
+      end
+
+      def self.get(offender_no)
```

Comment:
Wondering if this could be a bit more explicit e.g. "get_offender"? Someone new to the code might not find it immediately obvious.  This is only tiny though, so feel free to ignore

Reply:
it was originally `get_offender` so no objections to changing it back :)